frontend/app/src/main/java/com/chads/vanroomies/RecyclerData.java

```diff
@@ -0,0 +1,28 @@
+package com.chads.vanroomies;
+// Reference: https://www.geeksforgeeks.org/recyclerview-using-gridlayoutmanager-in-android-with-example/
+public class RecyclerData {
```

Comment:
Can we also change the name here to make it for Listings specifically since ChatFragment will likely need its own recyclerdata class too

Reply:
Yep!